Sources/Authgear.swift

```diff
@@ -27,6 +27,7 @@ struct AuthenticateOptions {
     let colorScheme: ColorScheme?
     let wechatRedirectURI: String?
     let page: AuthenticationPage?
+    let idTokenHint: String?
```

Comment:
I think we should not add idTokenHint to AuthenticateOptions. Instead of reusing createAuthenticateRequest, we should add createReauthenticateRequest.

Reply:
@louischan-oursky Removed and added `createReauthenticateRequest`, thanks!